src/css/header.css

```diff
@@ -28,3 +29,36 @@
     border-bottom: 2px solid var(--brand-green);
     opacity: 1;
 }
+
+/*A suggestion of header adjustment below*/
+
+
+/* .navigation {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+} */
+
+.navigation__logo {
+    margin-top: 26.91px;
```

Comment:
round these numbers, it makes no sense to have broken numbers for properties. I thought it would use rem instead of pixel.